src/components/Countdown/CountdownContainer.module.scss

```diff
@@ -0,0 +1,33 @@
+.container {
+    display: flex;
+    justify-content: center;
+    padding-right:20px;
+}
+
+.countdownContainer {
```

Comment:
I love the border and spacing of the countdown timer. When the timer expires, it should have the same border and spacing (bonus: if you're watching it expire, it won't jump around when it changes state).

Reply:
Thank you! Is this better? 

![Dark mode version of example with counter active](https://imgur.com/8Yn7GOo.png)
![Dark mode version of example with counter active](https://imgur.com/qaoCh3T.png)
![Light mode version of example with counter active](https://imgur.com/Km0ZERn.png)
![Light mode version of example with counter active](https://imgur.com/vafe4YA.png)

Reply:
Looks great! 🎉

Reply:
@djsauble Thank you 😃 I just noticed you approved the PR and I just submitted some last changes. I noticed that I forgot to specify the time zone. Thank you so much for your feedback! This is my first time attempting to contribute to an open source project and it has been a wonderful experience! 🎉